Assets/ObjectPool.cs

```diff
@@ -14,7 +14,8 @@ public class ObjectPool : MonoBehaviour
     public GameObject pooledObject;     //Object to pool and instantiate
     public int initialPoolSize;         //Number of pooledObjects that will be initialized into the pool
     private Stack<GameObject> pool;     //Stack that represents the object pool
-
+    [SerializeField]
+    private GameObject[] objectPrefabs;
```

Comment:
Decided to use an array instead of stack so I can retrieve any type of object in the pool